Source/Core/Core/FreeLookManager.h

```diff
@@ -54,6 +56,17 @@ class FreeLookController final : public ControllerEmu::EmulatedController
 
 private:
   ControllerEmu::Buttons* m_move_buttons;
+
+  ControllerEmu::SettingValue<double> m_pos_x;
+  double m_pos_last_x;
+
+  ControllerEmu::SettingValue<double> m_pos_y;
+  double m_pos_last_y;
+
+  ControllerEmu::SettingValue<double> m_pos_z;
+  double m_pos_last_z;
```

Comment:
I believe these are cast to float whenever they're used. If there's a reason they need to be doubles anyway I would add a comment explaining why.

Reply:
Numeric settings and also qt spin boxes all operate in terms of doubles but our Free Look code is in terms of floats.  I'll add a comment